CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
+<<<<<<< HEAD
```

Comment:
Looks like merging has gone wrong.

Reply:
Fixed in [5f1a755](https://github.com/Kampfkarren/selene/pull/565/commits/5f1a755b32acc8d36ed96c417eabaf81159018ee)